### PR TITLE
Improve docs and adjust setup.py

### PR DESCRIPTION
**Pull request by karras**
This is a simple and small PR to improve the readability of the README. In addition it aligns the license mention in `setup.py` with what's defined in the LICENSE file aside from adjusting some outdated information.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -55,7 +60,6 @@ b'\x01\x88?\x00'
 >>> mock_panel.stop()
 >>> mockthread.kill()
 >>> mockthread.join()
-
```
Comment (fujexo) [inline README.md:58]: This empty line is actually required for the doctest to pass

```
___________________________________________________________________ [doctest] README.md ____________________________________________________________________
053 >>> panel_control.connect()
054 >>> mockthread.start()
055 >>> panel_control.set_text("test01")
056 >>> panel_control.get_text()
057 'test01     '
058 >>> panel_control.get_serial_number(10)
059 b'\x01\x88?\x00'
060 >>> mock_panel.stop()
061 >>> mockthread.kill()
062 >>> mockthread.join()
Expected:
    ```
Got nothing

/home/fuj/repos/github.com/adfinis/sbb-fallblatt/README.md:62: DocTestFailure
```

File: README.md [hunk 2]
```diff
@@ -86,5 +90,4 @@ b'\x01\x88?\x00'
...
```
Comment (fujexo) [inline README.md:89]: This empty line is actually required for the doctest to pass

```
___________________________________________________________________ [doctest] README.md ____________________________________________________________________
053 >>> panel_control.connect()
054 >>> mockthread.start()
055 >>> panel_control.set_text("test01")
056 >>> panel_control.get_text()
057 'test01     '
058 >>> panel_control.get_serial_number(10)
059 b'\x01\x88?\x00'
060 >>> mock_panel.stop()
061 >>> mockthread.kill()
062 >>> mockthread.join()
Expected:
    ```
Got nothing

/home/fuj/repos/github.com/adfinis/sbb-fallblatt/README.md:62: DocTestFailure
```

**Review by fujexo — CHANGES_REQUESTED** [review]
Thanks for your contribution! I've added some requests regarding the doc tests.
Not sure why the GitHub Actions are currently not starting, maybe this resolves after pushing the requested changes.

File: README.md (hunk 2 above)
Comment (karras) [reply README.md]: Fixed, thanks!

File: README.md (hunk 1 above)
Comment (karras) [reply README.md]: Fixed, thanks!

**Comment by karras** [comment]
@fujexo I've integrated the requested changes, thanks for the pointers!

**Review by fujexo — APPROVED** [review]

**Pull request merged**